alternator/stats.cc

```diff
@@ -29,6 +29,8 @@ stats::stats() : api_operations{} {
 						                        seastar::metrics::description("Latency summary of an operation via Alternator API"), [this]{return to_metrics_summary(api_operations.name.summary());})(op(CamelCaseName)).set_skip_when_empty(),
             OPERATION(batch_get_item, "BatchGetItem")
             OPERATION(batch_write_item, "BatchWriteItem")
+            OPERATION(batch_get_item_batch_total, "BatchGetItemSize")
+            OPERATION(batch_write_item_batch_total, "BatchWriteItemSize")
             OPERATION(create_backup, "CreateBackup")
```

Comment:
@nyh why do we have macros in code?

Reply:
It's a macro mainly because of the usage of the `#` operator: `OPERATION(create_backup, "CreateBackup")` uses the `create_backup` twice - once to name a variable `api_operations.create_backup`, and once as a string `"create_backup"`. I could have done this without a macro, in a longer way:
```
operation(api_operations.create_backup, "create_backup", "CreateBackup")
```